handlers/main.yml

```diff
@@ -1,4 +1,3 @@
 ---
 - name: restart memcached
   service: name=memcached state=restarted
-  when: not memcached_install.changed
```

Comment:
once memcached is started when installed, restart memcached at end of playbook is required in order to memcached conf take effects